tests/ExcelProvider.Tests/ExcelProvider.Tests.fs

```diff
@@ -380,3 +381,12 @@ let ``HashHeader false with header removed``() =
     let expectedTime = new DateTime(1899, 12, 31, 8, 0, 0)
     row.Column5 |> should equal expectedTime
     row.Column6 |> should equal 100.0M
+
+// See https://github.com/fsprojects/ExcelProvider/issues/14
+[<Test>]
+let ``Can automatically coerce non-string cells in a column of string data to their string form`` () =
+    let file = MixedDataTypes()
+    let printTitles data = 
+        for (row:MixedDataTypes.Row) in data do 
+            sprintf "%s (%i)" row.Title ((int) row.Year) |> ignore
```

Comment:
One thing to note here; I believe, as written, a few other cases might cause throw InvalidCastException.

For example, if one of the values in the "year" column was a cell of type "Text". (I'd expect Double.Parse to resolve that) Or if a DateTime column had a cell as a number corresponding to an OA Date (DateTime.FromOADate). 

I played around with them a bit while working on this but wanted to keep this focused on just the issue in #14. I plan to make another PR with some of those additional assumptions.